Jenkinsfile_CNP

```diff
@@ -29,7 +29,15 @@ String notificationsChannel = '#sscs-tech'
 
 withPipeline("nodejs", product, component) {
 
-  before('securitychecks') {sh 'yarn test:audit'}
+  before('securitychecks') {
+    try {
+      sh 'yarn audit'
+    } catch (error) {
+      onMaster {
```

Comment:
let's remove onMaster to be able to confirm it works in the PR as expected, shall we?

Reply:
good point done !